src/libsync/vfs/cfapi/cfapiwrapper.cpp

```diff
@@ -58,6 +58,12 @@ void cfApiSendTransferInfo(const CF_CONNECTION_KEY &connectionKey, const CF_TRAN
         qCCritical(lcCfApiWrapper) << "Couldn't send transfer info" << QString::number(transferKey.QuadPart, 16) << ":" << cfExecuteresult << QString::fromWCharArray(_com_error(cfExecuteresult).ErrorMessage());
     }
 
+    // Once the download is finished, there is no need to do a last update of the progress status
+    if (offset + currentBlockLength == totalLength)
+    {
```

Comment:
style nitpick: brace should be on previous line

One day we'll have clang-format-diff checks on the CI side and I'll put my OCD to rest. ;-)

Reply:
Thanks.

I am not sure if the comment I have added is really needed here. What do you think ?

Reply:
Well, you felt the need for a comment so I guess something is needed for readability. Admittedly, in situations like this, I'm more a proponent of well named variables before comments (I keep comments as last resort). In the case at hands it'd give:
```cpp
const auto isDownloadFinished = ((offset + currentBlockLength) == totalLength);
if (isDownloadFinished) {
```

Note: not sure I'd always prescribe using parentheses on the whole expression like I did here, just felt more readable to me for this one somehow (probably the number of = on the line and the distance between = and == pushed me there, no rule to make out of that).

Reply:
This is indeed what I should have done at first.

I noticed that my comment was a sign of code not being good enough when reading back what I had written.